projects/admin-core/src/lib/shared/components/authorization-edit/authorization-edit.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, OnInit, OnDestroy, Input } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { AuthorizationRuleDecision, AuthorizationRuleGroup, GroupModel } from '@tailormap-admin/admin-api';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'tm-admin-authorization-edit',
+  templateUrl: './authorization-edit.component.html',
+  styleUrls: ['./authorization-edit.component.css'],
+  providers: [{
+    provide: NG_VALUE_ACCESSOR,
+    multi: true,
+    useExisting: AuthorizationEditComponent,
+  }],
+})
+export class AuthorizationEditComponent implements OnInit, OnDestroy, ControlValueAccessor {
+  private destroyed = new Subject();
+
+  private _onChange: (_: any) => void = () => undefined;
+  public value: AuthorizationRuleGroup[] = [];
+
+  private _groups: GroupModel[] = [];
+
+  public groupList: { name: string; used: boolean }[] = [];
+  public canAddRule = true;
+
+  @Input()
+  public set groups(value: GroupModel[]) {
+      this._groups = value;
+      this.updateGroupList();
+  }
+
+  private updateGroupList() {
+      this.groupList = this._groups.filter(a => !a.systemGroup).map(a => ({ name: a.name, used: this.value.find(b => b.groupName === a.name) !== undefined }));
+      this.canAddRule = this.groupList.find(a => !a.used) !== undefined;
+  }
+
+
+  constructor() { }
+
+  public ngOnInit(): void {
+  }
+
+  public ngOnDestroy() {
+      this.destroyed.next(null);
+      this.destroyed.complete();
+  }
+
+  public selectedChip = 'anonymous';
+
+  private updateValue(value: AuthorizationRuleGroup[], notify: boolean) {
+      this.value = value;
+      this.updateGroupList();
+
+      if (notify) {
+          this._onChange(value);
+      }
+
+      const anonymousGroup = this.value.find(a => a.groupName === 'anonymous');
```

Comment:
Good to convert these special values to `enum` so they can be reused and easily adjusted if API changes